docs/reference/api/connectivity/lorawan/lorawanconfig.md

```diff
@@ -1,4 +1,4 @@
-<h2 id="lorawan-configuration">Configuring Mbed LoRaWAN Stack</h2>
+<h2 id="lorawan-configuration">LoRaWAN Configuration</h2>
 
```

Comment:
We have this information in the configuration page: https://os-doc-builder.test.mbed.com/docs/development/reference/configuration-connectivity.html

Reply:
The information in configuration page is not useful to anyone. At least I can't read it or make sense of it. Can you @SeppoTakalo ? I would like to see the configuration page for LoRaWAN alongside API docs.

Reply:
@kjbracey-arm FYI

Reply:
@hasnainvirk 

"The information in configuration page is not useful to anyone. At least I can't read it or make sense of it."

The [config section overview page](https://os-doc-builder.test.mbed.com/docs/development/reference/configuration.html) tells users how to use `mbed compile --config -v` to get configuration parameters. Each of the pages within that section show the configuration parameters for those groupings. For example, we have platform configuration parameters, RTOS configuration parameters and storage configuration parameters. I would like to add some context above the list of parameters for each page, like we see with the [mesh configuration page](https://os-doc-builder.test.mbed.com/docs/development/reference/configuration-mesh.html). 

The only difference in content between the config page in your PR and the existing connectivity config page's LoRa content is that your PR doesn't include the lora.automatic-uplink-message function.

As your page doesn't add new information, I don't understand how you find your suggested addition useful but not the existing page.

Because platform, drivers, RTOS, connectivity, mesh and storage configuration parameters are all located within the configuration section at the request of engineering, it doesn't make sense for LoRa configuration to lie elsewhere.